core/payment/src/api/allocations.rs

```diff
@@ -202,14 +202,19 @@ pub async fn release_allocation_after(
 ) {
     tokio::task::spawn(async move {
         if let Some(timeout) = allocation_timeout {
-            let timestamp = timeout.timestamp() - Utc::now().timestamp();
-            let mut deadline = 0u64;
+            //FIXME when upgrading to tokio 1.0 or greater. The maximum duration of delay in tokio 0.2 is equal to max_duration(millis).
+            let max_duration = 1 << 36;
```

Comment:
It would be safer with explicit type. I'm not sure what type will be inferred here

Reply:
Ok, it's explicitly typed as i64 now